src/main/java/org/jboss/license/dictionary/LicenseDbStore.java

```diff
@@ -27,6 +27,14 @@
 import org.jboss.license.dictionary.model.License;
 import org.jboss.license.dictionary.model.LicenseAlias;
 import org.jboss.license.dictionary.model.LicenseApprovalStatus;
+import org.jboss.license.dictionary.model.LicenseDeterminationType;
+import org.jboss.license.dictionary.model.LicenseHintType;
+import org.jboss.license.dictionary.model.Project;
+import org.jboss.license.dictionary.model.ProjectEcosystem;
+import org.jboss.license.dictionary.model.ProjectVersion;
+import org.jboss.license.dictionary.model.ProjectVersionLicense;
+import org.jboss.license.dictionary.model.ProjectVersionLicenseCheck;
+import org.jboss.license.dictionary.model.ProjectVersionLicenseHint;
 import org.jboss.logging.Logger;
```

Comment:
Why not use slf4j?

Reply:
Logging is supported by WildFly Swarm using the typical JBoss-Logging systems. I will create a new PR in case we want to switch to slf4j (need to understand how to do it), after having collected the requirements :-)